### DISCUSSION TITLE
may i know how to install laragon in mac?

### DISCUSSION THREAD

**Discussion by ManojPadhiyar1993** [leokhoa/laragon#567] [Q&A] [answered]
may i know how to install laragon in mac?

**Comment by Pen-y-Fan** [comment, marked as answer]
Laragon is only for Windows.

On a Mac, I use [Laravel Valet](https://laravel.com/docs/10.x/valet) with [PHP Monitor](https://phpmon.app/), which is the closest I've found to Laragon.

Laravel Valet isn't only for Laravel, it has first and third-party support for other frameworks e.g.:

> Out of the box, Valet support includes, but is not limited to:

> [Laravel](https://laravel.com/), [Bedrock](https://roots.io/bedrock/), [CakePHP 3](https://cakephp.org/), [ConcreteCMS](https://www.concretecms.com/), [Contao](https://contao.org/en/), [Craft](https://craftcms.com/), [Drupal](https://www.drupal.org/), [ExpressionEngine](https://www.expressionengine.com/), [Jigsaw](https://jigsaw.tighten.co/), [Joomla](https://www.joomla.org/), [Katana](https://github.com/themsaid/katana), [Kirby](https://getkirby.com/), [Magento](https://magento.com/), [OctoberCMS](https://octobercms.com/), [Sculpin](https://sculpin.io/), [Slim](https://www.slimframework.com/), [Statamic](https://statamic.com/), Static HTML, [Symfony](https://symfony.com/), [WordPress](https://wordpress.org/), [Zend](https://framework.zend.com/),

> However, you may extend Valet with your own [custom drivers](https://laravel.com/docs/10.x/valet#custom-valet-drivers).

There is also [Laravel Herd](https://herd.laravel.com/), which uses Valet "under the hood", but comes complete with PHP Monitor equivalent, so it is an easier install.  I haven't tried it personally, but it does look good, the announcement was at Laracon 2023. If I was installing from scratch I would give Herd a try.

Other development tools, such as Node, MySQL workbench, etc., can be installed using [Homebrew](https://formulae.brew.sh/formula/).

Edit: fix links.

**Reply by ManojPadhiyar1993** [reply]
hi @Pen-y-Fan  thank you for reply i am new with mac os so it little difficult to manage
appreciate your reply

**Reply by ManojPadhiyar1993** [reply]
@Pen-y-Fan  i already have herd but when i try to run my core php project it not works

**Reply by ManojPadhiyar1993** [reply]
i just set up mamp :)

**Comment by benyrfg** [comment]
I finally found my setup: [Laravel Valet](https://laravel.com/docs/10.x/valet) + [PHP Monitor](https://phpmon.app/) + [DBngin](https://dbngin.com/)

**Reply by nubab4088** [reply]
@benyrfg does it work fully like laragon?will it affect the team work?

**Reply by benyrfg** [reply]
Yes, you can easily switch between PHP / DB versions, you have .test domain and local SSL certificate. I don't work in a team so I don't have the answer.
